### PR TITLE
Nostrum

### PR DESCRIPTION
**Pull request by trv6**

### CODE REVIEW

File: addons/Nostrum/variables.lua [hunk 1]
```diff
@@ -1,4 +1,4 @@
---[[Copyright © 2014-2015, trv
+--[[Copyright © 2014, trv
```
Comment (z16) [inline addons/Nostrum/variables.lua]: Why did you remove 2015 from here?

Comment (trv6) [inline addons/Nostrum/variables.lua]: Daylight... savings... time?
Fixed.

Comment (alphaONE2) [inline addons/Nostrum/variables.lua]: ROFL! Thank you, you just made my day!

Comment (z16) [inline addons/Nostrum/variables.lua]: Yeah, those can be a bitch :\
